src/test/resources/org/broadinstitute/hellbender/tools/walkers/qc/pileup/expectedPileup.txt

```diff
@@ -0,0 +1,238 @@
+1 200 N A B [Feature(s): [VC Unknown @ 1:199-200 Q232.46 of type=INDEL alleles=[GG*, G] attr={} GT=[], [VC Unknown @ 1:200 Q232.46 of type=SNP alleles=[G*, A] attr={} GT=[]] -130
```

Comment:
I'm struggling a lot with the test files, because there are a lot of them and I don't know how they are organized. I've already have problems implementing some tests without finding the correct reference. Do you have any suggestion, @akiezun? Which set of files could I use for the tests?